### PR TITLE
Added DiscoveredHost widget test

### PR DESCRIPTION
**Pull request by vijay8451**
- Added Discovered Hosts widget test around dashboard 
- Airgun PR: https://github.com/SatelliteQE/airgun/pull/311
- Test result:
```
========================================================================= test session starts ==========================================================================
platform linux -- Python 3.6.6, pytest-4.0.2, py-1.7.0, pluggy-0.8.1 -- /home/vijsingh/my_projects/myenv3/bin/python36
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/vijsingh/my_projects/PR_Proj/robottelo, inifile:
plugins: services-1.3.1, mock-1.10.0
collecting ... 2019-03-31 17:37:39 - conftest - DEBUG - BZ deselect is disabled in settings

collected 1 item                                                                                                                                                       

tests/foreman/ui_airgun/test_dashboard.py::test_positive_discovered_host PASSED                                                                                  [100%]

====================================================================== 1 passed in 60.02 seconds =======================================================================
```

### CODE REVIEW

File: tests/foreman/ui_airgun/test_dashboard.py [hunk 1]
```diff
@@ -372,3 +378,55 @@ def test_positive_content_host_subscription_status(session):
...
+    org = entities.Organization().create()
+    loc = entities.Location(organization=[org]).create()
+    name = gen_string('alpha')
+    ipaddress = gen_ipaddr()
+    macaddress = gen_mac(multicast=False)
+    model = gen_string('alpha', length=5)
+    host_name = 'mac{0}'.format(macaddress.replace(':', ''))
+    entities.DiscoveredHost().facts(json={
+        u'facts': {
+            u'name': name,
```
[the remark this answers is not in the document]
Comment (vijay8451) [reply tests/foreman/ui_airgun/test_dashboard.py]: Addressed in last commit.

File: tests/foreman/ui_airgun/test_dashboard.py [hunk 2]
```diff
@@ -14,6 +14,12 @@
 
 :Upstream: No
 """
+from fauxfactory import (
+    gen_string,
```
[the remark this answers is not in the document]
Comment (vijay8451) [reply tests/foreman/ui_airgun/test_dashboard.py]: Addressed in last commit.

**Comment by vijay8451** [comment]
- Added hosts_count assertion 
- resolved PR conflict 
- test result:
tests/foreman/ui_airgun/test_dashboard.py::test_positive_discovered_host PASSED                                                                                  [100%]

File: tests/foreman/ui_airgun/test_dashboard.py [hunk 3]
```diff
@@ -446,6 +451,58 @@ def test_positive_content_host_subscription_status(session):
...
+            u'memorysize_mb': 1000,
+            u'physicalprocessorcount': 2,
+        }
+    })
+
+    with session:
+        session.organization.select(org_name=org.name)
+        session.location.select(loc_name=loc.name)
+        values = session.dashboard.read('DiscoveredHosts')
+        assert len(values) == 2
```
Comment (oshtaier) [inline tests/foreman/ui_airgun/test_dashboard.py]: no-no, you have to check that there is only one value in a table, not the count of the values in your list

Comment (vijay8451) [reply tests/foreman/ui_airgun/test_dashboard.py]: @oshtaier nice catch .. It was a typo .
I have pushed the same in last commit .

**Pull request merged**